ratis-grpc/src/main/java/org/apache/ratis/grpc/server/GrpcLogAppender.java

```diff
@@ -594,7 +594,8 @@ private void installSnapshot(SnapshotInfo snapshot) {
     StreamObserver<InstallSnapshotRequestProto> snapshotRequestObserver = null;
     final String requestId = UUID.randomUUID().toString();
     try {
-      snapshotRequestObserver = getClient().installSnapshot(responseHandler);
+      snapshotRequestObserver = getClient().installSnapshot(getFollower().getName() + "-installSnapshot",
```

Comment:
We can add the `requestId` to the name of snapshotRequestObserver, to tell from different installSnapshot processes.